pandas/_libs/tslibs/parsing.pyx

```diff
@@ -33,6 +34,24 @@ from dateutil.parser import parse as du_parse
 from pandas._libs.tslibs.ccalendar import MONTH_NUMBERS
 from pandas._libs.tslibs.nattype import nat_strings, NaT
 
+cdef extern from "errno.h":
+    int errno
+
+cdef extern from "../src/parser/tokenizer.h":
```

Comment:
We’ve tried to keep the exposed surface between cython and C limited to np_datetime.  If this dep is strictly necessary, it may be worth putting there.

Reply:
We can do this.